modules/reference/pages/properties/cluster-properties.adoc

```diff
@@ -5160,6 +5176,24 @@ List of superuser usernames.
 
 ---
 
+=== tls_min_version
+
+The minimum TLS version that Redpanda supports.
+
+*Requires restart:* Yes
+
+*Optional:* No
+
+*Visibility:* `user`
+
+*Accepted values:* `v1.0`, `v1.1`, `v1.2`, `v1.3`
+
+*Type:* string
```

Comment:
```suggestion
*Type:* enum
```

Reply:
It's implemented as an enum of strings. For the end user it's a string.